Enum

chloeyutianyi opened the pull request.

Fleker commented:
It looks like you revised a few of the lint issues, but a few of them remain.

Fleker commented on peripherals/example.yaml:
```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```
It seems like this change may cause unit tests to fail as Emboss is currently not in the CI environment. If we revert this change and put together a separate `peripherals/example-emboss.yaml` file it should not cause any CI failures.

Fleker commented on peripherals-yaml/example.yaml:
```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```
It looks like you moved this file, but we still require the original. Can you copy this file back and make sure `peripherals/example.yaml` does not include the `imports`?

chloeyutianyi commented:
@googlebot I fixed it

On Tue, Aug 4, 2020 at 4:45 PM google-cla[bot] <notifications@github.com>
wrote:

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://urldefense.com/v3/__https://cla.developers.google.com/__;!!LIr3w8kk_Xxm!_cMkYhi0Dr9AM-ruMXtZ-_HrDp2eciOFkjGqzf0Gt7lxiI-iLYsLcDj882p_UVc$>
> to double check)? If these were authored by someone else, then they will
> need to sign a CLA as well, and confirm that they're okay with these being
> contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://urldefense.com/v3/__https://goto.google.com/prinfo/https*3A*2F*2Fgithub.com*2Fgoogle*2Fcyanobyte*2Fpull*2F216__;JSUlJSUlJQ!!LIr3w8kk_Xxm!_cMkYhi0Dr9AM-ruMXtZ-_HrDp2eciOFkjGqzf0Gt7lxiI-iLYsLcDj8BIQLRQM$>
> for more info*.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/216*issuecomment-668883847__;Iw!!LIr3w8kk_Xxm!_cMkYhi0Dr9AM-ruMXtZ-_HrDp2eciOFkjGqzf0Gt7lxiI-iLYsLcDj8vZgCU_8$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AHOX5245NLM5FES7X63NVYDR7CMTBANCNFSM4PTWQLDA__;!!LIr3w8kk_Xxm!_cMkYhi0Dr9AM-ruMXtZ-_HrDp2eciOFkjGqzf0Gt7lxiI-iLYsLcDj8-Q1giGA$>
> .
>

chloeyutianyi commented:
@googlebot I fixed it..

On Tue, Aug 4, 2020 at 5:32 PM google-cla[bot] <notifications@github.com>
wrote:

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://urldefense.com/v3/__https://cla.developers.google.com/__;!!LIr3w8kk_Xxm!4HKEPBr1N0vdbIRwnIrhPtqkzqnWNd-S26ayUf3KjN_P_M_qG2u-M-2GnE-1L8A$>
> to double check)? If these were authored by someone else, then they will
> need to sign a CLA as well, and confirm that they're okay with these being
> contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://urldefense.com/v3/__https://goto.google.com/prinfo/https*3A*2F*2Fgithub.com*2Fgoogle*2Fcyanobyte*2Fpull*2F216__;JSUlJSUlJQ!!LIr3w8kk_Xxm!4HKEPBr1N0vdbIRwnIrhPtqkzqnWNd-S26ayUf3KjN_P_M_qG2u-M-2GAzNZ7fk$>
> for more info*.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/216*issuecomment-668899906__;Iw!!LIr3w8kk_Xxm!4HKEPBr1N0vdbIRwnIrhPtqkzqnWNd-S26ayUf3KjN_P_M_qG2u-M-2GqoNvSYc$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AHOX526X3HQGOADN6PM4ENDR7CSAHANCNFSM4PTWQLDA__;!!LIr3w8kk_Xxm!4HKEPBr1N0vdbIRwnIrhPtqkzqnWNd-S26ayUf3KjN_P_M_qG2u-M-2GOdpIhtA$>
> .
>
